Blogs/resources/views/layouts/auth/main.blade.php

```diff
@@ -0,0 +1,5 @@
+@include('layouts.auth.header')
```

Comment:
- remove @include from layouts file
- ```layouts/``` should contain .blade files. No need to have folders